www/source/tutorials/build-your-own/define-processes.html.slim

```diff
@@ -0,0 +1,99 @@
+---
+title: Habitat - Define Processes
+---
+
+section
+ h1 Define Processes
+
+ p #{link_to 'Processes', 'https://12factor.net/processes'} are defined in the #{link_to '12-factor app document','https://12factor.net/'}  as running services or ephemeral tasks. Each of these processes are wrapped up in a small script which sets up a suitable app environment and invokes a command. By convention, a web application using the Ruby scaffolding will use the <code>web</code> process in the default implementation of the run hook. 
```

Comment:
I've removed the first line, but then added in some preamble so that we're not just dropping people straight into an example at the top of the page. You can let me know what you think after I've pushed my changes up.